_datasets/philadelphia-properties-and-assessment-history.md

```diff
@@ -8,9 +8,11 @@ maintainer: opadata@phila.gov
 maintainer_email: opadata@phila.gov
 maintainer_link: null
 maintainer_phone: null
-notes: "***OPA is currently updating its data files. Some of the information in the\
-  \ open data files below may not yet reflect the data used to calculate the tax year\
-  \ 2023 property value.***\r\n\r\nProperty characteristic and assessment history\
+notes: "***Some of the information in the open data files below may not yet reflect the\
+  \data used to calculate the most recent tax year’s property value. If you see missing\
+  \or incorrect info about your property, [use this form](https://phila.formstack.com/forms/property_feedback_form)\
+  \to contact OPA to report the issue.***\
```

Comment:
```suggestion
  \ data used to calculate the most recent tax year’s property value. If you see missing\
  \ or incorrect info about your property, [use this form](https://phila.formstack.com/forms/property_feedback_form)\
  \ to contact OPA to report the issue.***\
```

Have a feeling this lack of spacing just triggered mistaken quoting (e.g., the `\t` for tab, but the two invalid ones).  Maybe this?

Reply:
great!  Apologies, I'm just an open source contributor and city resident, so I'm unable to make those changes to your update. But I have a feeling that'll let this get up to the website cleanly! (if you can make them, or one of the primary maintainers) Have a great day!!